setup.py

```diff
@@ -302,13 +316,24 @@ class BuildFailed(Exception):
     pass
 
 
+def get_arch_specific_objects():
+    machine = cpuinfo.platform.machine()
+    if machine == 'x86_64':
+        return [S[:-1] + 'o' for S in glob("c-blosc/internal-complibs/zstd*/decompress/*amd64.S")]
+    elif machine == 'aarch64':
+        return [S[:-1] + 'o' for S in glob("c-blosc/internal-complibs/zstd*/decompress/*aarch64.S")]
+    return []
```

Comment:
It looks like this can be deleted, since it's not used?

Reply:
Ah, good catch! I notice that I added the same logic in the `run` method for the `ve_build_ext` class – removed in a86d4c20066c5b20bdc24df4146be0f02bb89a88.